### PR TITLE
WALL-30: Create SectionHeader

### PR DESCRIPTION
**Pull request by VasudevaVellanki4**
## Summary

Create a Section Header Component

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (code change that neither fixes a bug nor adds a feature)

### Linked Issues

NULL

## JIRA Issue(s)

- [JIRA Issue Link](https://wallaceproject.atlassian.net/browse/WALL-30)

## Checklist (check all)

- [ ] Automated tests have been added, or an explanation for why not
- [ ] Security implications and dependencies on linked features have been considered and explained
- _Provide an explanation if tests were not added. Convince the reviewers._

## Screenshots & Evidence
image

_Include any relevant screenshots, Videos, or links to documentation that provide additional context or proof of functionality._

### CODE REVIEW

**Review by raj-edition — APPROVED** [review]
Looks good

File: src/components/atoms/SectionHeader/SectionHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import { cn } from '@/lib/utils';
+
+import { Button } from '../Button';
+import { Heading } from '../Heading';
+import { SectionHeaderProps } from './SectionHeader.types';
+
```
Comment (manikangkan) [inline src/components/atoms/SectionHeader/SectionHeader.tsx:6]: Add jsdoc, check https://github.com/wallace-developments/wallace-frontend/pull/26/files

image

File: src/components/atoms/SectionHeader/SectionHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+}) => {
+  const defaultColor = variant === 'dark' ? 'inverted' : 'default';
+  const finalColor = color || defaultColor;
+
+  return (
+    <div
+      data-testid="section-header"
+      className={cn(
+        'flex flex-row justify-between items-center w-full',
+        'md:flex-col md:items-start',
```
Comment (manikangkan) [inline src/components/atoms/SectionHeader/SectionHeader.tsx:23]: image
image

Please check the figma ui once, we're changing the layout in `sm` for this

File: src/components/atoms/SectionHeader/SectionHeader.types.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { HeadingProps } from '../Heading';
+
+/**
+ * SectionHeaderProps interface
```
Comment (manikangkan) [inline src/components/atoms/SectionHeader/SectionHeader.types.ts:4]: let's update the jsdoc here as well

File: package.json [hunk 4]
```diff
@@ -23,7 +23,8 @@
     "install:node_modules": "yarn",
     "remove:node_modules": "rm -rf node_modules",
     "all:format": "yarn prettier:format && yarn eslint:format",
-    "prepare": "husky"
+    "prepare": "husky",
+    "checks": "yarn prettier:check && yarn lint && yarn test"
```
Comment (manikangkan) [inline package.json:27]: Revert this file please

**Comment by manikangkan** [comment]
https://github.com/wallace-developments/wallace-frontend/pull/38 Taken care

**Pull request closed without merging**